discord/commands/addMatchNight.ts

```diff
@@ -69,6 +70,10 @@ export async function addMatchNight(
       episode: i.options.find((e) => e.name == "episode")
         ?.value as number,
     });
+    getCurrenProbabilities(
+      i.options.find((e) => e.name == "season"),
+      i.options.find((e) => e.name == "episode"),
+    );
```

Comment:
Refactor the flow of the function so that the calculations do not influence the response of the command. A command cannot be unanswered for more than 15 min. I think the function call can be copied and pasted to the top level command in discord main.ts.